osu.Game/Overlays/Profile/Header/BottomHeaderContainer.cs

```diff
@@ -131,7 +131,11 @@ private void updateDisplay(User user)
             tryAddInfo(FontAwesome.Brands.Discord, user.Discord);
             tryAddInfo(FontAwesome.Brands.Skype, user.Skype, @"skype:" + user.Skype + @"?chat");
             tryAddInfo(FontAwesome.Brands.Lastfm, user.Lastfm, $@"https://last.fm/users/{user.Lastfm}");
-            tryAddInfo(FontAwesome.Solid.Link, websiteWithoutProtcol, user.Website);
+            tryAddInfo(FontAwesome.Solid.Link, websiteWithoutProtocol, user.Website);
+
+            // Hide the container to prevent adding unnecessary padding if it has no children other than the NewLine
+            if (!bottomLinkContainer.Children.Skip(1).Any())
+                bottomLinkContainer.Hide();
```

Comment:
This check is really brittle since you're only checking the count. I'd say this at least needs to check if the only child element is a `NewLineContainer` for this to be solid. You could also refactor `tryAddInfo` slightly and do something like:

```csharp
bool anyInfoAdded = false;
anyInfoAdded |= tryAddInfo(FontAwesome.Solid.MapMarker, user.Location);
anyInfoAdded |= tryAddInfo(OsuIcon.Heart, user.Interests);
// ...
if (!anyInfoAdded)
    bottomLinkContainer.Hide();

private bool tryAddInfo(IconUsage icon, string content, string link = null)
{
    if (string.IsNullOrEmpty(content)) return false;
    // existing body
    return true;
}
```

Reply:
Did this by checking if the only child is a `NewLineContainer`, since it looks less cluttered IMO. Let me know if that's good enough.

Reply:
I'm fine with the type check, I'm less fine with the double negation you now have in there. Reads better as
```csharp
if (bottomLinkContainer.Children.All(child => child is NewLineContainer))
```
IMO.

Reply:
believe me or not, I was wondering how I could improve it and completely forgot that `.All()` exists, looks like I need a bit of sleep. fixed